caas/kubernetes/provider/bootstrap.go

```diff
@@ -1348,6 +1392,17 @@ func (c *controllerStack) controllerContainers(jujudCmd, controllerImage string)
 				SubPath:   cloudconfig.FileNameBootstrapParams,
 				ReadOnly:  true,
 			},
+			{
+				Name:      application.CharmVolumeName,
```

Comment:
we don't need this, the controller uses the same oci image as the init container that places this here.

The pebble command should just use `/opt/pebble`